scripts/components/CollectionList.js

```diff
@@ -34,29 +34,12 @@ class Row extends Component {
     }
   }
 
-  recordField(displayField) {
-    const {record} = this.props;
-    if (record.hasOwnProperty(displayField)) {
-      const field = record[displayField];
-      if (typeof field === "string") {
-        return field;
-      } else if (typeof field === "object") {
-        return JSON.stringify(field);
-      } else {
-        return String(field);
-      }
-    } else if (displayField === "__json") {
-      return <code>{JSON.stringify(cleanRecord(record))}</code>;
-    }
-    return "<unknown>";
-  }
-
   render() {
     const {bid, cid, record, displayFields} = this.props;
     return <tr onDoubleClick={this.onDoubleClick.bind(this)}>
       {
         displayFields.map((displayField, index) => {
-          return <td key={index}>{this.recordField(displayField)}</td>;
+          return <td key={index}>{recordField(displayField, this.props.record)}</td>;
```

Comment:
Nit: `record` is already destructured and available, no need to get it from `this.props`.